kornia/geometry/warp/homography_warper.py

```diff
@@ -71,35 +148,9 @@ def precompute_warp_grid(self, src_homo_dst: torch.Tensor) -> None:
               The homography assumes normalized coordinates [-1, 1] if
               normalized_coordinates is True.
          """
-        self._warped_grid = self.warp_grid(src_homo_dst)
-
-    def warp_grid(self, src_homo_dst: torch.Tensor) -> torch.Tensor:
-        r"""Compute the grid to warp the coordinates grid by the homography/ies.
+        self._warped_grid = warp_grid(self.grid, src_homo_dst)
 
-        Args:
-            src_homo_dst (torch.Tensor): Homography or homographies (stacked) to
-              transform all points in the grid. Shape of the homography
-              has to be :math:`(1, 3, 3)` or :math:`(N, 1, 3, 3)`.
-              The homography assumes normalized coordinates [-1, 1] if
-              normalized_coordinates is True.
-
-        Returns:
-            torch.Tensor: the transformed grid of shape :math:`(N, H, W, 2)`.
-        """
-        batch_size: int = src_homo_dst.shape[0]
-        device: torch.device = src_homo_dst.device
-        dtype: torch.dtype = src_homo_dst.dtype
-        # expand grid to match the input batch size
-        grid: torch.Tensor = self.grid.expand(batch_size, -1, -1, -1)  # NxHxWx2
-        if len(src_homo_dst.shape) == 3:  # local homography case
-            src_homo_dst = src_homo_dst.view(batch_size, 1, 3, 3)  # Nx1x3x3
-        # perform the actual grid transformation,
-        # the grid is copied to input device and casted to the same type
-        flow: torch.Tensor = transform_points(
-            src_homo_dst, grid.to(device).to(dtype))  # NxHxWx2
-        return flow.view(batch_size, self.height, self.width, 2)  # NxHxWx2
-
-    def forward(  # type: ignore
+    def forward(
```

Comment:
regarding the  `# type: ignore`, did they correct the issue of mypy with the forward method? I will be very happy :)

Reply:
I don't really know what do you mean :)
I removed it because jit gave errors because of it

Reply:
I do not remember why, but mypy hates the forward function. If it passes the test it is fine.

Reply:
great! good to see that they fixed this :)